src/configs/FirebaseConfig.js

```diff
@@ -35,6 +35,7 @@ const FirebaseConfig = (props) => {
     if (dynamicLink?.url?.includes('postPrivateId=')) return handlePrivatePost(dynamicLink);
     if (dynamicLink?.url?.includes('communityName')) return handleCommunityPage(dynamicLink);
     if (dynamicLink?.url?.includes('users?username=')) return getUserProfile(dynamicLink?.url);
+    if (dynamicLink?.url?.includes('profile/')) return getUserProfileV2(dynamicLink?.url);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new condition added to check if the URL includes 'profile/' seems fine. However, it's important to ensure that `getUserProfileV2` function is properly defined and handles all edge cases since it's being used here.

```diff
+    if (dynamicLink?.url?.includes('profile/')) return getUserProfileV2(dynamicLink?.url);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->